examples/read_all_input_channels.py

```diff
@@ -0,0 +1,54 @@
+import curses
+import time
+from pydualsense import *
+
+
+dualsense = pydualsense()
+dualsense.init()
+
+
+while dualsense.states is None:
+    print("no states found yet")
+    print(f"epoch: {time.time():.0f}")
+    time.sleep(0.5)
+
+
+stdscr = curses.initscr()
+curses.noecho()
+curses.cbreak()
+
+
+while True:
```

Comment:
could you add some option to gracefully close the curses display and stop the example? maybe by pressing q ?